helm/hpcc/templates/_helpers.tpl

```diff
@@ -1244,6 +1244,46 @@ dali data
 {{- end -}}
 {{- end -}}
 
+{{/*
+A template to generate the standard app.kubernetes.io labels
+
+root name(k8s application name) component(component within the application, can be same as app) instance 
+
+https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+----------------------------+-------------------------------------------------------------+----------
+| Label                     | Description                                                 | Example |
+----------------------------+-------------------------------------------------------------+----------
+app.kubernetes.io/name       The name of the application                                   mysql
+app.kubernetes.io/component  The component within the architecture                         database
+app.kubernetes.io/instance   A unique name identifying the instance of an application      mysql-abcxzy
+app.kubernetes.io/version    The current version of the application                        5.7.21
+app.kubernetes.io/part-of    The name of a higher level application this one is part of    wordpress
+app.kubernetes.io/managed-by The tool being used to manage the operation of an application helm
+app.kubernetes.io/created-by The controller/user who created this resource 	               controller-manager
+*/}}
+{{ define "hpcc.addStandardLabels" }}
```

Comment:
should chomp space at start:
```
{{- define "hpcc.addStandardLabels" }}
```

it's debatable whether should at end too I suppose, but I think correct to leave template adding newline at start by not chomping whitespace at end ( -}} )
Which means when this template is used, you want | indent, not | nindent.